bin/e2e-test-runner

```diff
@@ -1,16 +1,49 @@
 #!/bin/bash
 set -e
 
+
+export DEBUG=1
+export NO_RESTART_LOOP=1 
+export CYPRESS_BASE_URL=http://localhost:8080
+export OPT_OUT_CAPTURE=1
+export SECURE_COOKIES=0
+export PRIMARY_DB=clickhouse
+export SKIP_SERVICE_VERSION_REQUIREMENTS=1
+export KAFKA_URL=kafka://kafka:9092
+export CLICKHOUSE_DATABASE=posthog_test
+export TEST=1
+export CLICKHOUSE_SECURE=0
+export E2E_TESTING=1
+export DATABASE_URL=postgres://localhost:5432/posthog_e2e_test
+export SECRET_KEY=e2e_test
+export EMAIL_HOST=email.test.posthog.net
+export SITE_URL=test.posthog.net 
+export REDIS_URL=redis://localhost/
+
+nc -z localhost 9092 || ( echo -e "\033[0;31mKafka isn't running. Please run\n\tdocker compose -f ee/docker-compose.ch.arm64.yml up zookeeper kafka clickhouse db redis\nI'll wait while you do that.\033[0m" ; bin/check_kafka_clickhouse_up )
+wget -nv -t1 --spider 'http://localhost:8123/' || ( echo -e "\033[0;31mClickhouse isn't running. Please run\n\tdocker compose -f ee/docker-compose.ch.arm64.yml up zookeeper kafka clickhouse db redis.\nI'll wait while you do that.\033[0m" ; bin/check_kafka_clickhouse_up )
+
+
+
 dropdb --if-exists posthog_e2e_test
 createdb posthog_e2e_test
-DEBUG=1 DATABASE_URL=postgres://localhost:5432/posthog_e2e_test python manage.py migrate &&
-DEBUG=1 DATABASE_URL=postgres://localhost:5432/posthog_e2e_test python manage.py setup_dev &
+python manage.py shell --command="from ee.clickhouse.client import sync_execute;sync_execute('drop database if exists posthog_test');sync_execute('create database posthog_test')"
```

Comment:
Are we using `posthog_test` and `posthog_e2e_test` or should we just drop one?

Reply:
oh sorry, that's clickhouse

Reply:
using posthog_test for clickhouse so this line is correct

Reply:
(comments would be welcome)